src/audio/volume/volume_hifi4.c

```diff
@@ -21,7 +21,7 @@ LOG_MODULE_DECLARE(volume_hifi4, CONFIG_SOF_LOG_LEVEL);
 
 #include "volume.h"
 
-#ifdef VOLUME_HIFI4
+#if SOF_USE_HIFI(4, VOLUME) || SOF_USE_HIFI(5, VOLUME)
```

Comment:
Not sure if we want to add the HIFI5 case in same patchset, but no big harm either,

Reply:
`SOF_USE_HIFI(5,... )` is now required because I already added HIFI5 in common.h. So, without `SOF_USE_HIFI(5,... )` here, the linker would fail to compile any platform that HAS_HIFI5 with the error "volume code not found".

> I already added HIFI5 in common.h.

Was that premature? If it was premature then feel free to remove it.

Reply:
Add HIFI5 have below advantage:
1. no error, since it is || relation.
2. If add new platform build, like PTL and NVL in near future, there is no HIFI5 implementation for some modules, with this adding, it can make sure still use HIFI4 as best for each module.
3. it is also a reminder for later module HIFI5 adding.